recipes/openpiv/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set version = "0.20.4" %}
+
+package:
+  name: openpiv
+  version: {{ version }}
+
+source:
+  fn: openpiv-{{ version }}.tar.gz
+  url: https://github.com/OpenPIV/openpiv-python/archive/{{ version }}.tar.gz
+  sha256: 4921c30d5f6ee5234ba69ec09ab4e2e6a4913a5a3ac99c8e6bd1a09e7ba7e9ba
+  
+build:
+  number: 0
+  skip: true  # [py3k]
+  script: 
+    - python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy x.x
+    - scipy
+    - cython
+    - scikit-image
+    - nose
```

Comment:
Please drop any dependencies that are not needed during the build. Probably can drop `nose`, `scikit-image`, and `scipy`.